Google/resource-manager/folder/locals.tf

```diff
@@ -3,8 +3,13 @@ locals {
   folders = try(yamldecode(local.folders_yml), {})
   folder_components       = try(lookup(local.folders, "components", {}), lookup(local.folders, "components"))
```

Comment:
We don't actually have `specs` https://github.com/mesoform/Multi-Cloud-Platform#adapter-top-level-attributes